client/rest-high-level/src/main/java/org/elasticsearch/client/IndexLifecycleClient.java

```diff
@@ -139,6 +140,34 @@ public AcknowledgedResponse stopILM(StopILMRequest request, RequestOptions optio
                 AcknowledgedResponse::fromXContent, emptySet());
     }
 
+    /**
+     * Get the status of index lifecycle management
+     * See <a href="https://fix-me-when-we-have-docs.com">
+     * the docs</a> for more.
+     *
+     * @param request the request with user defined timeouts.
+     * @param options the request options (e.g. headers), use {@link RequestOptions#DEFAULT} if nothing needs to be customized
+     */
+    public StatusILMResponse StatusILM(TimedRequest request, RequestOptions options) throws IOException {
```

Comment:
I think these should be `LifecycleManagementStatusResponse`, rather than using the ILM in method/class names

Also, the method name is `StatusILM` and we don't capitalize any of our method names

Reply:
> Also, the method name is StatusILM and we don't capitalize any of our method names

oops didn't meant to capitalize the method name. (kinda surprised that check style didn't catch that). will fix. 

> I think these should be LifecycleManagementStatusResponse, rather than using the ILM in method/class names

I tend to agree, but was matching the start/stopILM here:  https://github.com/elastic/elasticsearch/tree/index-lifecycle/x-pack/protocol/src/main/java/org/elasticsearch/protocol/xpack/indexlifecycle 

Should I rename those too ?